### DISCUSSION TITLE
Podman only

### DISCUSSION THREAD

**Discussion by jeffmaury** [redhat-developer/vscode-openshift-tools#3385] [Q&A]
Do we need to be connected to a cluster ? I can't figure out how to run locally against podman.

**Comment by sbouchet** [comment]
if odo cli can run against podman without being connected to a cluster, IDE extensions should as well.

**Comment by datho7561** [comment]
Yes, you should be able to run on podman without being connected to a Kubernetes/OpenShift cluster. Right click on the component in the OpenShift sidebar, then select "Start Dev on Podman":

![running-on-podman](https://github.com/redhat-developer/vscode-openshift-tools/assets/22376627/79d9d5a3-936d-4234-a955-e2d345676282)
